psi4/src/psi4/dfocc/occ_iterations.cc

```diff
@@ -275,11 +275,6 @@ void DFOCC::occ_iterations() {
             break;
         }
 
-        if (wfn_type_ != "DF-OLCCD") {
-            if (rms_wog < tol_grad && biggest_mograd < mograd_max) break;
```

Comment:
I only had a quick look at the list of fixes, but just out of curiosity, why was this removed?

Reply:
To fix #1143, which was interfering with my tests.